main.cpp

```diff
@@ -386,30 +386,30 @@ struct KitchenRoom
     void removeFromTable(bool isOnTable);
 };
 
-KitchenRoom::KitchenRoom()
+Room::Room()
 {
     std::cout << "Kitchen room object constructed!" << '\n';
 }
 
-KitchenRoom::~KitchenRoom()
+Room::~Room()
 {
     std::cout << "Kitchen room object destructed!" << '\n';
 }
 
-bool KitchenRoom::putRadioOnTableAndSwitchOn()
+bool Room::putRadioOnTableAndSwitchOn()
 {
-    this->kitchen.table = true;    
-    if(this->radio.switchOn() && this->kitchen.table)
+    this->kitchen.tableIsOccupied = true;    
+    if(this->radio.switchOn() && this->kitchen.tableIsOccupied)
     {
         std::cout << "Radio put on table!" << '\n';
         return true;
     }
     return false;
 }
 
-void KitchenRoom::removeFromTable(bool onTable)
+void Room::removeFromTable(bool onTable)
 {
-    if(onTable) this->kitchen.table = false;
+    if(onTable) this->kitchen.tableIsOccupied = false;
```

Comment:
write single-line `if` expression bodies on a separate line to make the code easier to read
```
if( condition )
    expression
```    
- [ ] revise